packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -473,9 +473,23 @@ const Services = WebexPlugin.extend({
       return Promise.resolve(priorityUrl || priorityUrlObj.priorityUrl);
     }
 
+    if (this._getCatalog().isReady) {
+      return url ?
+        Promise.resolve(url) :
+        Promise.reject(new Error(
+          `services: service '${name}' was not found after waiting`
+        ));
+    }
+
     const catalog = this._getCatalog();
     const groupsToCheck = ['limited', 'signin', 'postauth'];
 
+    if (catalog.isReady) {
```

Comment:
do we need this check here since we already check if catalog is ready 2 lines above?